### PR TITLE
Support Drupal 8.7 and Openshift 3.11

### PR DESCRIPTION
**Pull request by singularo**
Drupal 8.7 was dying on the cmi import because of a couple of missing things.
Redis support has some empty array elements which don't work with 3.11.
Snuck in a fix for the wordpress generator as well.

### CODE REVIEW

**Comment by singularo** [comment]
Note - related PR - https://github.com/universityofadelaide/openshift-client/pull/35 - required for 3.11 support.

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "universityofadelaide/shepherd-drupal-scaffold": "~2.0",
         "wikimedia/composer-merge-plugin":               "^1.4",
         "drush/drush":                                   "9.*",
-        "drupal/core":                                   "8.5.*",
+        "drupal/core":                                   "^8.7",
```
Comment (pingers) [inline composer.json]: 8.6?

Comment (pingers) [reply composer.json]: Or @beta

**Comment by pingers** [comment]
Just need to go back to Drupal core 8.6 till 8.7 is out in a couple weeks.

**Comment by singularo** [comment]
Back to 8.6

**Pull request merged**